### PR TITLE
[155] Style the Contact Us Page using CSS

### PR DESCRIPTION
**Pull request by Mayowa2020**
Updates on the html and addition of new cs
Screenshot (184)
Screenshot (185)
Screenshot (186)
Screenshot (187)
Screenshot (188)
Screenshot (189)

s files for my contact page.

### CODE REVIEW

**Comment by Ireti203** [comment]
Weldone @Mayowa2020 You currently have an open PR that is yet to be merged. Whenever you have an open PR, It is safer you ensure it gets merged before opening another.

**Review by adaugochi — CHANGES_REQUESTED** [review]
Remove spaces from images name e.g `facebook logo.jpg, facebook logo.png, gmail logo.png, gmail logo.jpg` etc. Also, these images appear twice, the differences in the extension(png or jpg). why not just use one of them and remove the rest.

Meanwhile...well done. Keep up with the good work

**Comment by adaugochi** [comment]
Remove everything pertaining to product on this PR

File: submissions/BukkyOyetimehin/HTML-Forms/contact.html [hunk 1]
```diff
@@ -1,28 +1,74 @@
...
-		    <input type="text" name="yourfullname" id="yourfullname" placeholder="your full name.."> <br><br>
-		    <label for="email">Email Address*: </label> <br>
-		    <input type="email" name="email" id="email" placeholder="your email address.."> <br><br>
-		    <label for="subject">Subject*:</label> <br>
-		    <input type="text" name="subject" id="subject"> <br><br>		    	
-		    <label for="comments">Message:</label> <br>
-		    <textarea name="comments" id="comments" cols="40" rows="10" placeholder="Write something.."></textarea> <br>
-		    <input type="submit" value="Submit" name="submit">
-		</fieldset>
-    </form>
```
Comment (Mayowa2020) [inline submissions/BukkyOyetimehin/HTML-Forms/contact.html:26]: This is not part of the file.

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**